descriptions.go

```diff
@@ -262,7 +264,14 @@ func buildDependencyTree(contents []IssueWizardItem, conf *Configuration, credsm
 	var result []IssueWizardItem                         // to return
 	resultMap := map[CredentialTypeIdentifier]struct{}{} // to keep track of credentials already put in the result slice
 	for i := len(depTree) - 1; i >= 0; i-- {
+		var credIDs []CredentialTypeIdentifier
```

Comment:
You've added order to the wizards now. Currently a test sometimes fails because the test assumes order. Is order really necessary? Couldn't we better change the test not to depend on the order instead of changing production code?